src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -67,6 +67,10 @@ def __init__(self):
         self.port_index_namespace = {}
         self.namespace_db_map = Namespace.get_namespace_db_map(self.db_conn)
 
+        self.statedb = mibs.init_db()
```

Comment:
This will not work for multi-asic database as this will only initialize the host database.
Could use the  self.db_conn and connect to state_db using:
Namespace.connect_all_dbs(self.db_conn, mibs.STATE_DB)